pkg/cosign/common.go

```diff
@@ -27,12 +27,15 @@ import (
 )
 
 // TODO(jason): Move this to an internal package.
-func FileExists(filename string) bool {
+func FileExists(filename string) (bool, error) {
```

Comment:
What do other semver folks think about this change? It's technically breaking, but I can't find any usage of it on GitHub.

Reply:
Good catch. Also, to Jason's point, I think this should be in an internal package, as this behaviour doesn't need to be public to users. It's trivial to implement if they need to. That would prevent us from running into issues with breaking semver.

Reply:
Yeah, if we're going to change the interface we might as well move it internal at the same time to prevent future issues. WDYT about making the change?

Reply:
Sure. I can make a commit to that effect. Thanks

Reply:
+1